patch/0000-net-Fix-netdev-adjacency-tracking.patch

```diff
@@ -0,0 +1,1302 @@
+From 962236dd11d64946c9d0c80982b1f75adb1c0780 Mon Sep 17 00:00:00 2001
+From: ps407411 <ps407411@contoso.com>
+Date: Wed, 4 Sep 2019 10:22:48 +0000
+Subject: [PATCH] net: Fix netdev adjacency tracking
+
+
+Note: This patch is no longer required in kernel version >=4.10
+
+Refer to below patches which have been combined into single patch.
+
+https://github.com/torvalds/linux/commit/67b62f98a1de962277b60d77c0c208b76867dbae#diff-3ce2f1b11ea5a4b9c88ef0b4159bd481
+https://github.com/torvalds/linux/commit/0f524a80ff35af8a7664d7661d948107da142e04#diff-3ce2f1b11ea5a4b9c88ef0b4159bd481
+https://github.com/torvalds/linux/commit/f1170fd462c67c4ae2f20734566d94e0f8f62f69#diff-3ce2f1b11ea5a4b9c88ef0b4159bd481
+https://github.com/torvalds/linux/commit/1a3f060c1a47dba4e12ac21ce62b57666b9c4e95#diff-3ce2f1b11ea5a4b9c88ef0b4159bd481
+https://github.com/torvalds/linux/commit/790510d99f39cee7f275d001aa5024032ed9bb48#diff-3ce2f1b11ea5a4b9c88ef0b4159bd481
+
+https://github.com/torvalds/linux/commit/453d39329ad03f9f6d93ec6d0d57bf7a1e2a20c7#diff-66286e176927fcbf186d37fce71bc94e
+https://github.com/torvalds/linux/commit/e0e79c8e74b08976d9b45e52b704b9228c6965c7#diff-598fcd5f9f85a0fea3a53a7022974457
+
+https://github.com/torvalds/linux/commit/b3208b2024c9089106df52ae25ebf39068d6f9fc#diff-ce347f03cd1a8602476d0774dd8dde24
+
+https://github.com/torvalds/linux/commit/1cd127fc7d3a6d6a0fc9f9cca47ca3a16ee79679#diff-69464fa7779eabc57449a324cd70bbcb
+
+https://github.com/torvalds/linux/commit/dd82364c3ab93a96bb1e45d22106a4b1ea4bef39#diff-5a740fc1e7dea601fffed2ecbbb8e7c8
```

Comment:
https://github.com/torvalds/linux/commit/dd82364c3ab93a96bb1e45d22106a4b1ea4bef39#diff-5a740fc1e7dea601fffed2ecbbb8e7c8


according to the description, it is not related to the ipv6 nexhop feature. why is this needed?

Reply:
@lguohan As part of this PR there are 2 commits for 2 issues:
1. IPv6 host and subnet anycast route in VRF to be configured with nhop interface as actual netdev instead of VRF L3MDEV : Commit : https://github.com/Azure/sonic-linux-kernel/pull/101/commits/f6dab4a42bd128bb2bbe75ab9f067456b70b1389
2. Fix for Unregister_netdev VRF obsreved and kernel hangs during vrf delete : https://github.com/Azure/sonic-linux-kernel/pull/101/commits/09715a31b6f7948542c1fd5327d4d424ab3f2970

1st commit is fix for IPv6 host and subnet anycast route
2nd commit is fix for Kernel hang observed during vrf deletion. 2nd commit comprises single patch which is merged from multiple commits made to linux kernel.

Accidentally I created single PR for both issues. If required, I can create separate PR for above 2 issues. Please let me know.
Note: Both above commits are necessary for VRF feature.

Reply:
yes, please separate. thanks.